pkg/deploy/generator/templates.go

```diff
@@ -63,7 +63,9 @@ func (g *generator) rpTemplate() *arm.Template {
 	}
 
 	if g.production {
-		t.Resources = append(t.Resources, g.pip(), g.lb(), g.vmss(), g.lbAlert())
+		t.Resources = append(t.Resources, g.pip(), g.lb(), g.vmss(),
+			g.lbAlert(50.0, 3, "rp-availability-alert", "PT5M", "PT1H"), // 3/3 backends down for 30min or 2/3 backends down for 45min in the last 1h
```

Comment:
I think if all 3 backends are dead for 30 min we might have a bigger problem and no need to wait for 30 min?